dev-docs/bidders/nexverse.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: bidder
+title: Nexverse
+description: Prebid Nexverse Bidder Adapter
+biddercode: nexverse
+tcfeu_supported: false
+dsa_supported: true
+gvl_id: none
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: false
+userId: all
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: true
+pbjs: true
+pbs: false
+prebid_member: false
+multiformat_supported: will-bid-on-one
+ortb_blocking_supported: false
+privacy_sandbox: no
+sidebarType: 1
+---
+
+
+# Nexverse Adapter
```

Comment:
Actually - just noticed that you're starting with a level-1 header here. Adapters need to start at level 3. Please update this header and all the ones below.